### PR TITLE
Sort project sensitivity

### PR DESCRIPTION
**Pull request by frankyma**
Used to sort by the sensitivity words alphabetically.  Now use CASE to map it to a numerical value to be sorted

### CODE REVIEW

File: test/project.e2e-spec.ts [hunk 1]
```diff
@@ -507,6 +507,65 @@ describe('Project e2e', () => {
     );
   });
 
+  it.only('List of projects sorted by Sensitivity', async () => {
```
Comment (CarsonF) [inline test/project.e2e-spec.ts:510]:
```suggestion
  it('List of projects sorted by Sensitivity', async () => {
```

File: src/components/project/project.service.ts [hunk 2]
```diff
@@ -526,7 +526,15 @@ export class ProjectService {
...
     const projectSortMap: Partial<Record<typeof input.sort, string>> = {
       name: 'lower(prop.value)',
+      sensitivity: 'sensitivityValue',
     };
+
+    const sensitivityCase = `case prop.value 
+    when 'High' then 3
+    when 'Medium' then 2
+    when 'Low' then 1
+    end as sensitivityValue`;
```
Comment (CarsonF) [inline src/components/project/project.service.ts:536]: Can a `case` expression not be used in an `orderby` statement?

**Review by CarsonF — CHANGES_REQUESTED** [review]

File: src/components/project/project.service.ts (hunk 2 above)
Comment (frankyma) [reply src/components/project/project.service.ts:536]: not that i've seen, it's usually inside a RETURN or a WITH in the examples.  I played around with it in ORDERBY and couldn't get it to work.

File: test/project.e2e-spec.ts (hunk 1 above)
Comment (frankyma) [reply test/project.e2e-spec.ts]: fixed

Comment (frankyma) [reply test/project.e2e-spec.ts]: fixed

File: src/components/project/project.service.ts [hunk 3]
```diff
@@ -537,10 +545,13 @@ export class ProjectService {
         q
           .match([
             node('node'),
-            relation('out', '', sort),
+            relation('out', '', sort, { active: true }),
```
Comment (CarsonF) [inline src/components/project/project.service.ts:548]: Is this necessary? If so it should be applied to every list

Comment (michaelmarshall) [reply src/components/project/project.service.ts:548]: yes. When we traverse over property relationships we must always test for active true in order to return only non-deleted properties.

**Pull request merged**